var/spack/repos/builtin/packages/glint2/package.py

```diff
@@ -0,0 +1,60 @@
+from spack import *
+
+class Glint2(CMakePackage):
+    """Regridding/Coupling library for GCM + Ice Sheet Model"""
+
+    homepage = "https://github.com/citibeth/icebin"
+    url         = "https://github.com/citibeth/icebin/tarball/v0.1.0"
+
+    version('0.1.0', '1c2769a0cb3531e4086b885dc7a6fd27')
+    version('glint2', git='https://github.com/citibeth/icebin.git', branch='glint2')
+
+    variant('python', default=True, description='Build Python extension (requires Python, Numpy)')
+    variant('coupler', default=True, description='Build the GCM coupler (requires MPI)')
+    variant('pism', default=False, description='Build coupling link with PISM (requires PISM, PETSc)')
+
+    extends('python', when='+python')
+
+#    depends_on('everytrace+fortran', when='~coupler')
+    depends_on('everytrace+mpi+fortran')
+
+    depends_on('python@3:', when='+python')
+    depends_on('py-cython', when='+python')
+    depends_on('py-numpy', when='+python')
+
+    depends_on('cgal')
+    depends_on('gmp')
+    depends_on('mpfr')
+
+    depends_on('mpi', when='+coupler')
+    depends_on('pism@glint2~python', when='+coupler+pism')
+    depends_on('petsc@3.4.5~superlu-dist', when='+coupler+pism')
```

Comment:
Clearly there was a problem at some point.  Next time I build IceBin with
PISM (which could be a while), I can find out more specifically what
happens without `~superlu-dist`.  Maybe the problem has been fixed by now.

On Wed, Aug 17, 2016 at 5:07 AM, Denis Davydov <notifications@github.com>
wrote:

> In var/spack/repos/builtin/packages/glint2/package.py:
>
> > +    extends('python', when='+python')
> > +
> > +#    depends_on('everytrace+fortran', when='~coupler')
> > +    depends_on('everytrace+mpi+fortran')
> > +
> > +    depends_on('python@3:', when='+python')
> > +    depends_on('py-cython', when='+python')
> > +    depends_on('py-numpy', when='+python')
> > +
> > +    depends_on('cgal')
> > +    depends_on('gmp')
> > +    depends_on('mpfr')
> > +
> > +    depends_on('mpi', when='+coupler')
> > +    depends_on('pism@glint2~python', when='+coupler+pism')
> > +    depends_on('petsc@3.4.5~superlu-dist', when='+coupler+pism')
>
> why specifically ~superlu-dist? Is there a problem here if petsc is build
> with it?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/LLNL/spack/pull/1545/files/19f6d14c88c52cdbe5239323ee76c4d049b2b13d#r75087168>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB1cd4G2KQPJ5JB8vNbd5BTwbicdNK1jks5qgs85gaJpZM4JmFjy>
> .
>